### DISCUSSION TITLE
Did not plan route via bicycle path on dyke for moped

### DISCUSSION THREAD

**Discussion by anastasiia936** [osmandapp/OsmAnd#11622] [Maps Related]
In the Netherlands, between Lelystad and Enkhuizen, is a dyke, de Markerwaarddijk. This is a highway. There’s also a bicycle path beside it that could be used for moped. 
The program sees this way but does not plane route via this way for a moped on 4 wheels, allowed and capable of Going 45 km/hour. (BROMMOBIEL).
![image](https://user-images.githubusercontent.com/80623554/117279445-63db1280-ae6a-11eb-8d21-54f1b0e0d02d.png)
Start point: 52.694828, 5.278867
Finish point:52.527425, 5.434393

**Comment by AlexVeltrop** [comment]
to clear up my project..
I'd like the naviagtion programme to use the bicyclepath to navigate form starting point to finnish point / vica versa.
So i can use the path for my Brommobiel.

**Comment by scaidermern** [comment]
OsmAnd doesn't have a moped routing profile. So which routing profile did you use?

**Comment by AlexVeltrop** [comment]
The exact name of the profile:

Mini car brommobiel.

 

 

Alex Veltrop.

Koekoeksplein 4,

1021 VB, Amsterdam, Nederland.

0031652241017 Mobile

 ***@***.***> ***@***.***

 

NL75 TRIO 0379 4319 39

FRA42 2004 1010 0911 9836 2N03 057

**Comment by AlexVeltrop** [comment]
I made a new profile, Brommobiel..

A bike on 4 wheels allowed and capable going 45 km/hour.

Alowed on N??? roads up to 80/km hour.

Unless blocked by the sign C09. With an exeption of a undersighn OB 55

I did send that profile to OSMand on a few occaisions.

 

 

Alex Veltrop.

Koekoeksplein 4,

1021 VB, Amsterdam, Nederland.

0031652241017 Mobile

 ***@***.***> ***@***.***

 

NL75 TRIO 0379 4319 39

FRA42 2004 1010 0911 9836 2N03 057

**Reply by vshcherb** [reply]
There is no specific profile for "Bronmobiel" https://github.com/osmandapp/OsmAnd-resources/blob/master/routing/routing.xml. So basically you use Car profile but with a name "Bronmobiel" and that doesn't change anything

**Comment by vshcherb** [comment]
Well technically the profile name doesn't matter much, navigation profile of OsmAnd routing that what matters a lot. If you didn't modify or didn't create your routing.xml you might be using either 'car' or 'bike' profile which could be incorrect in your case. So in that sense moped profile needs to be created.
